### ISSUE TITLE
Ingrain not up to date with Gen 6 (not sure if intended or not)

### ISSUE THREAD

**Issue by Forrester885** [BuffelSaft/pokeemerald#352]
"Ghost Pokémon under the effects of Ingrain are now able to switch out. However, they still cannot be forced to switch by moves that switch the target out." -Bulbapedia

**Comment by BuffelSaft** [comment]
Yeah there's still a bunch of old effects that haven't been updated, will be fixed.
